editor/components/writing-flow/index.js

```diff
@@ -201,6 +202,25 @@ class WritingFlow extends Component {
 		}
 
 		if ( ! isNav ) {
+			const activeElement = document.activeElement;
+
+			// Set right before the meta+a combination can be pressed.
+			if ( is.primary( event ) ) {
+				this.isFullySelected = isFullySelected( activeElement );
+			}
+
+			if ( is.primary( event, 'a' ) ) {
+				// In the case of contentEditable, we want to know the earlier value
+				// because the selection will have already been set by TinyMCE.
+				if ( activeElement.isContentEditable ? this.isFullySelected : isFullySelected( activeElement ) ) {
+					onMultiSelect( first( blocks ), last( blocks ) );
+					event.preventDefault();
+				}
+
+				// Set in case the meta key doesn't get released.
```

Comment:
I feel this comment could be expanded to say what is being set. This reads as a sentence fragment and I'm not really sure what it means.

Reply:
I tried to improve it but unsure if it addressed the issue.

Reply:
That totally works actually, I get it now 😄